lib/primitive/test/spec/Cardano/Wallet/Primitive/Types/UTxOSpec.hs

```diff
@@ -402,8 +399,7 @@ deriving anyclass instance Function AssetName
 deriving anyclass instance CoArbitrary TokenPolicyId
 deriving anyclass instance Function TokenPolicyId
 
-instance CoArbitrary TxIn where
-    coarbitrary = coarbitraryTxIn
+deriving anyclass instance CoArbitrary TxIn
```

Comment:
Note that this relies on the default implementation of `coarbitrary`, available for types that are instances of `Generic`. (See https://hackage.haskell.org/package/QuickCheck-2.14.3/docs/Test-QuickCheck.html#v:coarbitrary)

Reply:
:+1: